torch/csrc/jit/passes/onnx.cpp

```diff
@@ -226,7 +226,94 @@ void BlockToONNX(
 
   // Finally, visit all nodes in the graph
   for (auto node : old_block->nodes()) {
-    if (node->kind() == prim::PythonOp) {
+    if (node->kind().is_caffe2()) {
```

Comment:
for readability - extract as a subfunction?

Reply:
we keep using this node->kind().is_onnx() or node->kind().is_caffe2() pattern. We can add it in a separate PR. :-)